sdk/nodejs/api/client.gen.ts

```diff
@@ -1353,6 +1515,12 @@ Defaults to './Dockerfile'.
 
   /**
    * Retrieves this directory plus a new file written at the given path.
+   * @param path Location of the written file.
+   * @example /file.txt
+   * @param contents Content of the written file.
+   * @example Hello world
```

Comment:
When you have more than one param, it create an example per param...

![image](https://user-images.githubusercontent.com/1186424/212038549-68b067bf-aba8-41dc-8b4c-8099af0d5aab.png)

Reply:
Ok I think I must change the format for the function to have only one example that groups the function call instead of an example by parameters.
I'll work on it.

Reply:
Fixed in https://github.com/dagger/dagger/pull/4367/commits/2a0ebdce4d802f8666d5963ca8963641169fd915

Reply:
same comment as here : https://github.com/dagger/dagger/pull/4367#discussion_r1069081894